### DISCUSSION TITLE
Two More Questions

### DISCUSSION THREAD

**Discussion by Jay123210599** [staxrip/staxrip#1293] [Q&A]
1. I cut a selection of a video with 100 frames, and I want to make an image sequence out of it with all frames, along with a gif/animated webp/apng file. How do I do that?
2. Whenever I open a video, it makes a folder with the video contents inside. How do I get the tool to stop doing that or is it necessary?

**Comment by Dendraspis** [comment]
1. Using `Tools » Animation » ...` for the Animation stuff and and `Imagewriter()` for the images:  https://avisynthplus.readthedocs.io/en/latest/avisynthdoc/corefilters/imagewriter.html
2. Depending on the source and selected settings it might be needed, Check out the `Options` to see if you can deselect some of the demuxings.

**Reply by Jay123210599** [reply]
I don't see animation here. And where do I put in Imagewriter? Do I have to download it first or is it already in there? How do I stop demuxing videos?
![Screenshot (419)](https://github.com/staxrip/staxrip/assets/141977822/d8a00680-c8b8-49ed-a1ca-50305020beb9)

**Reply by Dendraspis** [reply]
Sorry, I meant `Apps » Animation » ...`

> And where do I put in Imagewriter?

It's a filter. Double-click on the left filter list.

> How do I stop demuxing videos?

Check out `Options » Video`
